tests/decentralized_test.go

```diff
@@ -119,14 +117,17 @@ var _ = Describe("kairos decentralized k8s test", Label("decentralized-k8s"), fu
 			out, err := vm.Sudo("cat /run/cos/live_mode")
 			Expect(err).To(HaveOccurred(), out)
 			if isFlavor(vm, "alpine") {
-				Eventually(func() string {
-					out, _ = vm.Sudo("sudo cat /var/log/kairos/agent.log")
-					return out
-				}, 20*time.Minute, 1*time.Second).Should(
-					Or(
-						ContainSubstring("Configuring k3s-agent"),
-						ContainSubstring("Configuring k3s"),
-					), out)
+				// Skip for now as agent doesn't log anymore as it cannot behave both as a one-off and a daemon
```

Comment:
mmm not really getting this, do we stop logging out?

Reply:
yep, kairos-agent is now launched as a one-off script to avoid getting a crashed service. So we cant log directly like we did with the supervise daemon.